src/main/java/uk/gov/hmcts/reform/document/DocumentUploadClientApi.java

```diff
@@ -49,15 +49,17 @@ public DocumentUploadClientApi(
     public UploadResponse upload(
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
         @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuth,
+        @RequestHeader("user-id") String userId,
```

Comment:
could this user-id be a constant? its twice in this file?

Reply:
Ok, I'll extract it.